pkg/dohdec-cli/lib/cli.js

```diff
@@ -180,6 +181,7 @@ For more debug information:
       ecsSubnet: this.argv.ecsSubnet,
       ecs: this.argv.ecs,
       dnssec: this.argv.dnssec,
+      dnssecCheckingDisabled: this.argv.dnssecCd,
```

Comment:
the `dnssecCd` doesn't look right there?

Reply:
Good catch! Will fix it

Reply:
Fixed:

```
$ node bin/dohdec.js --dnssec --dnssecCheckingDisabled
domain (rrtype)> dnssec-failed.org
[
  {
    name: 'dnssec-failed.org',
    type: 1,
    TTL: 300,
    data: '96.99.227.255'
  },
  {
    name: 'dnssec-failed.org',
    type: 46,
    TTL: 300,
    data: 'A RSASHA1 2 300 1670251862 1668782762 44973 dnssec-failed.org. FlsTSg98nrXHmX5TLfORtA5r1DzyWojyTc6Dl81X4ssm6R5LKwUEin1D/8/MJoGRe7QKFpdL3VB//wDoilUiCUGoZDAqT/uF2Mi89PkOzyfIAy3Uv59nWvZJSfI369So/E991zLmZDekXjNTrJV43UtWhw5Wn2gim7LlX/Fsiog='
  }
]
```